code/modules/mob/living/carbon/human/human.dm

```diff
@@ -311,6 +311,29 @@
 	<BR>"}
 	show_browser(user, dat, name, "mob[name]")
 
+/mob/living/carbon/human/auto_observed(mob/dead/observer/observer)
```

Comment:
Please codedoc this, it's not very obvious what it does out there

Reply:
It should inherit the docs from `/mob/living/carbon/proc/auto_observed()` (at least in VSCode), but you are right.
I'll add something to explain this one better.